[Docs] Fix typos, improve, update at Get Started page (#5587)
huggingface/diffusers: docs/source/en/installation.md

</jax>
</frameworkcontent>

## Install with conda

> After activating your virtual environment, with `conda` (maintained by the community):

Reviewer: ```suggestion
After activating your virtual environment, you can also install with `conda`:
```

Author: Now, is conda maintained officially rather than by the community?

Reviewer: Good question, I don't think we officially maintain this but @patrickvonplaten or @sayakpaul would know more. 

If it is community maintained, then no need to make any changes :)
